modules/api/php/provisioners/projectimagesrowprovisioner.class.inc

```diff
@@ -66,7 +66,7 @@ class ProjectImagesRowProvisioner extends DBRowProvisioner
              LEFT JOIN files_qcstatus qc
                ON (f.FileID = qc.FileID)
              LEFT JOIN Project project
-               ON (c.ProjectID = project.ProjectID)
+               ON (s.ProjectID = project.ProjectID)
```

Comment:
This shouldn't be in this PR. Do you need to rebase?

Reply:
same answer as above regarding the base branch